### PR TITLE
enable load integration tests

### PR DESCRIPTION
**Pull request by baude**
fix bug where multiple images can be loaded and have to be able to handle the return of multiple names

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: test/e2e/load_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ var _ = Describe("Podman load", func() {
 	)
 
 	BeforeEach(func() {
-		Skip(v2fail)
+		//Skip(v2fail)
```
Comment (giuseppe) [inline test/e2e/load_test.go:23]: could we drop completely this line?

Comment (baude) [reply test/e2e/load_test.go:23]: yes ofc my mistake

**Comment by baude** [comment]
fixed

**Pull request merged**